QuickBytes/src/main/java/com/quickbytes/model/Combo.java

```diff
@@ -0,0 +1,20 @@
+package com.quickbytes.model;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+@Entity
+public class Combo {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long comboId;
+	
+	@OneToMany //fk
```

Comment:
This is saying one combo can have multiple requests?

Reply:
One combo can have multiple request  combos which are different from requests